sdks/python/apache_beam/runners/dask/transform_evaluator.py

```diff
@@ -83,21 +180,20 @@ def key(item):
 
     def value(item):
       k, v = item
-      return k, [elm[1] for elm in v]
+      return k, [defenestrate(elm[1]) for elm in v]
 
     return input_bag.groupby(key).map(value)
 
 
 class Flatten(DaskBagOp):
-  def apply(self, input_bag: OpInput) -> db.Bag:
+  def apply(self, input_bag: t.List[db.Bag]) -> db.Bag:
     assert type(input_bag) is list, 'Must take a sequence of bags!'
     return db.concat(input_bag)
 
 
 TRANSLATIONS = {
     _Create: Create,
     apache_beam.ParDo: ParDo,
-    apache_beam.Map: Map,
```

Comment:
Could you explain why Map was removed in this PR? Doesn't seem to be hurting anything based on testing, just curious

Reply:
I found via experiments that this was never exercised! `Map` translates to a composite of `ParDo` every time.